### PR TITLE
MOB-2074 - Update ULW name and appearance in the list

### PR DESCRIPTION
**Pull request by Oleg-Pecheneg**

### CODE REVIEW

File: unstoppable-ios-app/domains-manager-ios/Entities/Mock/MockEntitiesFabric.swift [hunk 1]
```diff
@@ -92,7 +92,8 @@ extension MockEntitiesFabric {
         static func mockMPC(hasRRDomain: Bool = true) -> WalletEntity {
             let address = "0x12313123"
             let domains = Domains.mockDomainsDisplayInfo(ownerWallet: address)
-            let udWallet = UDWallet.createMPC(address: address, 
+            let udWallet = UDWallet.createMPC(address: address,
+                                              aliasName: "Lite Wallet",
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Entities/Mock/MockEntitiesFabric.swift:96]: Lets use here the new `mpcWalletDefaultName`

File: unstoppable-ios-app/domains-manager-ios/Extensions/Extension-String+Preview.swift [hunk 2]
```diff
@@ -1232,6 +1232,7 @@ extension String {
         static let removeMPCWalletPullUpSubtitle = "REMOVE_MPC_WALLET_PULL_UP_SUBTITLE"
         static let mpcWalletMessagingUnavailableMessage = "MPC_WALLET_MESSAGING_UNAVAILABLE_MESSAGE"
         static let mpcWalletSigningUnavailableErrorMessage = "MPC_WALLET_SIGNING_UNAVAILABLE_ERROR_MESSAGE"
+        static let mpcWalletDefaultName = "MPC_WALLET_DEFAULT_NAME"
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-ios/Extensions/Extension-String+Preview.swift:1235]: Nobody knows what default is) How about 'mpcWalletShortName' -- speaks for itself

File: unstoppable-ios-app/domains-manager-iosTests/Helpers/TestableUDWalletsService.swift [hunk 3]
```diff
@@ -63,7 +63,10 @@ final class TestableUDWalletsService: UDWalletsServiceProtocol {
         throw TestableGenericError.generic
     }
     
-    func addOrUpdateMPCWallet(_ wallet: UDWallet) throws { }
+    func createMPCWallet(ethAddress: HexAddress,
+                         mpcMetadata: MPCWalletMetadata) throws -> UDWallet {
+        UDWallet.createMPC(address: ethAddress, aliasName: "Lite Wallet", mpcMetadata: mpcMetadata)
```
Comment (rommex) [inline unstoppable-ios-app/domains-manager-iosTests/Helpers/TestableUDWalletsService.swift:68]: Again, please use the string constant

**Review by rommex — APPROVED** [review]
With some suggestions

File: unstoppable-ios-app/domains-manager-ios/Extensions/Extension-String+Preview.swift (hunk 2 above)
Comment (Oleg-Pecheneg) [reply unstoppable-ios-app/domains-manager-ios/Extensions/Extension-String+Preview.swift:1235]: This is specifically the string that will be used as default name for the newly added MPC wallet. Default makes a lot of sense here. 
On the other hand mpc wallet short name implies that there's long and there's short name. MPC product name is not the same is mpc wallet name. 
And mpcWalletShortName might not scale well when product name will be changed.

**Pull request merged**